### PR TITLE
[14.0][ADD] account_move_import_queue_context

### PR DESCRIPTION
**Pull request by toita86**

### CODE REVIEW

**Review by amh-mw — CHANGES_REQUESTED** [review]
LGTM, but cover with unit test?

**Comment by toita86** [comment]
Just added, let me know if I'm missing something.

**Comment by francesco-ooops** [comment]
@amh-mw good for you?

File: account_move_import_queue_context/tests/test_account_move_import_queue_context.py [hunk 1]
```diff
@@ -0,0 +1,12 @@
+from odoo.tests.common import TransactionCase
+
+
+class TestJobPrepareContext(TransactionCase):
+    def test_job_prepare_context_before_enqueue_keys(self):
+        model = self.env["base"].create({})
```
Comment (HekkiMelody) [inline account_move_import_queue_context/tests/test_account_move_import_queue_context.py]: **question**(non-blocking): do you need this `create`? Couldn't you just do `self.env["base"]._job_prepare_context_before_enqueue_keys()` ?

**Review by HekkiMelody — APPROVED** [review]
Code review, LGTM

File: account_move_import_queue_context/tests/test_account_move_import_queue_context.py (hunk 1 above)
Comment (amh-mw) [reply account_move_import_queue_context/tests/test_account_move_import_queue_context.py]: I concur the create is unnecessary for methods decorated `@api.model`.

**Review by amh-mw — APPROVED** [review]

**Comment by francesco-ooops** [comment]
@pedrobaeza good to go?

**Comment by pedrobaeza** [comment]
I feel here there's a missing piece, as with this module, you can't import directly with `queue_job`. I also don't understand why going to such complex architecture, and more when Odoo has improved the import in future versions.

**Pull request closed without merging**